README.md

```diff
@@ -1,9 +1,9 @@
 <div align="center">
   
-![Preview](https://github.com/pushqrdx/dotfiles/raw/master/screenshots/vim.png)
```

Comment:
The read me should be adding info about the variant instead of replacing the original